src/apdu/dispatcher.c

```diff
@@ -73,7 +73,6 @@ int apdu_dispatcher(const command_t *cmd) {
                 // so we will just test for existence or not of data
                 return io_send_sw(SW_WRONG_DATA_LENGTH);
             } else if (cmd->p1 == SIGN_TX_P1_DONE && cmd->data) {
-
```

Comment:
Shouldn't we add a comment saying "nothing to do"? Or simply explain why there's nothing to do.

Reply:
Actually this will never run, since it's tested above. This was a series of  if-else-if but i condensed on a single if, i didn't clean this up properly, but it's been removed.